samples/app-cache-meetings/nodejs/src/components/app-cache-tab.tsx

```diff
@@ -0,0 +1,99 @@
+// <copyright file="app-cache-tab.tsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import { useState } from "react";
+import React from "react";
+import * as microsoftTeams from "@microsoft/teams-js";
+
+/// </summary>
+/// function logitem
+/// </summary>
+function logItem(action: string, actionColor: string, message: string) {
+    const newItem =
+        "<span style='font-weight:bold;color:" +
+        actionColor +
+        "'>" +
+        action +
+        "</span> " +
+        message +
+        "</br>";
+    return newItem;
+}
+
+/// </summary>
+/// In beforeUnloadHandler using setItems and readyToUnload callback function
+/// </summary>
+const beforeUnloadHandler = (
+    readyToUnload: () => void) => {
+    readyToUnload();
```

Comment:
you need to add a logItem here that shows tab was unloaded

Reply:
![image](https://user-images.githubusercontent.com/8000588/211117762-3c8d655c-102c-4a53-8a11-352f41d31eb0.png)

Reply:
Comment Resolved :
<img width="590" alt="image" src="https://user-images.githubusercontent.com/109147706/211612501-db003a62-505c-4de6-b78f-c32105210be0.png">

Reply:
Update all the images/screenshots. They are outdated